README.md

```diff
@@ -37,6 +37,38 @@ General help with the software and the website is also available on the IRC
 channel at [irc.esper.net#cytube](http://webchat.esper.net/?channels=cytube)
 during US daytime hours.
 
+Docker
```

Comment:
I would maybe put the docker instructions in a separate file (e.g. docs/docker.md) and then add a link to that from the top of https://github.com/calzoneman/sync/wiki/CyTube-3.0-Installation-Guide in case people want to use docker instead of following the usual installation instructions.